### ISSUE TITLE
Diagrammi delle sequenze

### ISSUE THREAD

**Issue by Andrea-Augello** [Andrea-Augello/SviluppoSW#26]
Essendo da modificarsi nel file astah non si possono fare i branch altrimenti si scassa tutto.
Per questo motivo non si può lavorare contemporaneamente o uno dei due che lavora al file perde tutte le modifiche fatte, quindi bisogna prestare molta attenzione.
Se non saltano fuori nuovi casi d'uso la quota di diagrammi da fare a testa è 5.

- [ ] Crea Prenotazione
	- [ ] InserisciDettagliRicetta
	- [ ] ScegliRegimeVisita
	- [ ] PrenotaVisitaSSN
	- [ ] PrenotaVisitaALPI
- [ ] Modifica Prenotazione
	- [ ] ScegliPrenotazione
	- [ ] SpostaPrenotazione
	- [ ] EliminaPrenotazione
	- [ ] ConfermaModifica
- [ ] Visualizza FSE
	- [ ] VisualizzaStoricoVisite
	- [ ] StampaCartellaClinica
	- [ ] VisualizzaCartellaClinica
- [ ] Effettua Autenticazione
	- [ ] AutenticaPersonale
	- [ ] AutenticaPaziente
	- [ ] RegistraPaziente
	- [ ] AutorizzazionePaziente
- [ ] Inserisci Dettagli Visita
- [ ] Notifica Prenotazione

**Comment by Frankie690II** [comment]
Quindi aggiungo i diagrammi di sequenza al file DiagrammiUML.asta e poi gli altri continuano da quel file? Giusto per capirci ed evitare di fare casini

**Comment by Andrea-Augello** [comment]
Esatto, non si possono fare i branch o esplode tutto perché è un file
binario.
I metodi che vai chiamando inseriscili prima come metodi della classe, così
poi facendo il diagramma delle classi non ce ne perdiamo
